backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/model/mapper/SubscriptionModelMapper.java

```diff
@@ -22,13 +25,27 @@
 @Mapper
 public interface SubscriptionModelMapper {
 
+	final EmbeddedWrappers embeddedWrappers = new EmbeddedWrappers(false);
```

Comment:
In my opinion, interfaces shouldn't have fields unless they are constants.

I think you should define an abstract mapper and move this field (along with the `wrap()` method below) into that new abstract class.

Reply:
See my new comments about using an abstract class vs an interface.